api/chain.go

```diff
@@ -466,3 +475,34 @@ func (a *API) chainBlockByHashHandler(msg *apirest.APIdata, ctx *httprouter.HTTP
 	}
 	return ctx.Send(convertKeysToCamel(data), apirest.HTTPstatusOK)
 }
+
+// POST /chain/organizations/filter/page/<page>
+// returns a list of organizations paginated by the given page
+func (a *API) chainOrganizationsFilterPaginated(msg *apirest.APIdata, ctx *httprouter.HTTPContext) error {
+	// get organizationId from the request body
+	var organizationId string
+	if err := json.Unmarshal(msg.Data, &organizationId); err != nil {
+		return ErrCantParseOrgID.WithErr(err)
+	}
+	// get page
+	var err error
+	page := 0
+	if ctx.URLParam("page") != "" {
+		page, err = strconv.Atoi(ctx.URLParam("page"))
+		if err != nil {
+			return ErrCantParsePageNumber.WithErr(err)
+		}
+	}
+	page = page * MaxPageSize
+	// get matching organization ids from the indexer
+	matchingOrganizationIds := a.indexer.EntityList(MaxPageSize, page, organizationId)
```

Comment:
Other endpoints return 204 http status code (with an error associated to it) when no results are found. I don't know if this is the expected behaviour here.

Reply:
I also have the same doubts about the following endpoints. We have already opened an issue with something similar https://github.com/vocdoni/interoperability/issues/16.